db/models/comments.model.js

```diff
@@ -1,10 +1,10 @@
-const db = require('../connection')
+const db = require('../connection');
 
 exports.fetchCommentsByArticle = (article_id) => {
     return doesArticleExist(article_id)
         .then((articleExists) => {
             if (!articleExists) {
-                res.status(404).send({ err: 404, msg: `Article not found for article ID: ${articleId}` });
+                throw { status: 404, message: `Article not found for article ID: ${article_id}` };
```

Comment:
This should be a promise rejct instead of a throw and our controller will pass it onto our error handlers.